justfile

```diff
@@ -0,0 +1,11 @@
+default:
```

Comment:
Perhaps we could use yarn/npm scripts instead of adding just tooling.

```
"scripts": {
    ...
    "nix-guardian": "./scripts/mprocs-nix-guardian.sh",
    "nix-gateway": "./scripts/mprocs-nix-gateway.sh",
    "mprocs": "mprocs -c ./mprocs.yml"
 }
```

Reply:
Even more, I propose we replace the current dev scripts. They are not useful
https://github.com/fedimint/ui/pull/86/commits/24e753afa4109b43d0ba05e3fdb22c1f0c7da784